src/components/AppliedFilters/AppliedFilters.tsx

```diff
@@ -72,10 +72,18 @@ const AppliedFilters = ({ aggregations }: { aggregations: Aggregation[] }) => {
   }
 
   if (!tagSetData.length) return null
+
+  // 'From' date filter should appear before 'to' date.
+  const sortedTagSetData = [...tagSetData].sort((a, b) => {
```

Comment:
are there only ever two fields here? it might be a little easier to debug if we can explicitly see greater than/less than and how that corresponds to order